pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -229,7 +234,7 @@ func storageProvisionerKey(sc *storagev1.StorageClass) string {
 
 var storageClassToProvisionerKeyMapper = map[string]func(sc *storagev1.StorageClass) string{
 	"pxd.openstorage.org": func(sc *storagev1.StorageClass) string {
-		//https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-shared-pvcs/
+		// https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-shared-pvcs/
```

Comment:
All these links are broken

Reply:
yeah since this tends to change maybe I could just add a comment that portworx docs served as the guideline?

Reply:
I also think so